submissions/tests/test_team_api.py

```diff
@@ -261,6 +261,45 @@ def test_create_submission_for_team_error_creating_individual_submission(self, m
         self.assertEqual(TeamSubmission.objects.count(), 0)
         self.assertEqual(Submission.objects.count(), 0)
 
+    def test_get_submissions_from_other_teams(self):
+        # Make a team submission with default users, under TEAM_1
+        self._make_team_submission(
+            attempt_number=1,
+            course_id=COURSE_ID,
+            item_id=ITEM_1_ID,
+            team_id=TEAM_1_ID,
+            status=None,
+            create_submissions=True
+        )
+        # Check against TEAM_2 with 2 additional user IDs added (that don't have a submission)
+        team_ids = [
+            self.anonymous_user_id_map[student] for student in [
+                self.user_1, self.user_2, self.user_3, self.user_4
+            ]
+        ] + ['55555555555555', '666666666666666666']
+        external_submissions = team_api.get_submissions_from_other_teams(
```

Comment:
would you mind tossing a `with self.assertNumQueries(1):` call around this line?